File: Programing EF DbContext/AfterCompletingChapter7/Model/Person.cs
using System.ComponentModel.DataAnnotations;
using System.Collections.Generic;

namespace Model
{
  [Table("People")]
  public class Person
  {
    public Person()
    {
      Address = new Address();
      Info = new PersonalInfo
      {
        Weight = new Measurement(),
        Height = new Measurement()
      };
    }

    public int PersonId { get; set; }
    [ConcurrencyCheck]
    public int SocialSecurityNumber { get; set; }
    [MinLength(3)]
    public string FirstName { get; set; }
    [MaxLength(10, ErrorMessage= "Dude! Last name is too long! 10 is max.")]
    public string LastName { get; set; }
    public Address Address { get; set; }
    public PersonalInfo Info { get; set; }

    public string FullName
    {
      get { return FirstName + " " + LastName; }
    }

    public List<Lodging> PrimaryContactFor { get; set; }
    public List<Lodging> SecondaryContactFor { get; set; }
    [Required]
    public PersonPhoto Photo { get; set; }
    public List<Reservation> Reservations { get; set; }
  }
}

File: Programming EF Code First/chapter3/Chapter 3 VB/Fluent/Model/Person.vb
Namespace Model
  Public Class Person
	Public Sub New()
	  Address = New Address()
	  Info = New PersonalInfo With {.Weight = New Measurement(), .Height = New Measurement()}
	End Sub

	Public Property PersonId() As Integer
	Public Property SocialSecurityNumber() As Integer
	Public Property FirstName() As String
	Public Property LastName() As String
	Public Property Address() As Address
	Public Property Info() As PersonalInfo
  End Class
End Namespace
